src/pybind/mgr/restful/module.py

```diff
@@ -251,7 +256,7 @@ def serve(self):
                 self._serve()
                 self.server.socket.close()
             except CannotServe as cs:
-                self.log.warn("server not running: {0}".format(cs.message))
+                self.log.warn("server not running: %s", cs)
```

Comment:
I personally prefer the ```.format()``` formatting.

Reply:
i am not changing line for no reason: `Exception` in py3 does not have the `message` property. and i think using the first param as format string is well documented by `logging.debug()`, moreover, it's simpler and shorter.

Reply:
ah yes...my mistake.